src/shell/views/Shell/Shell.less

```diff
@@ -24,18 +24,38 @@
   height: 100vh;
   overflow: hidden;
   display: grid;
-  grid-template-columns: 60px 1fr;
+  grid-template-columns: 160px 1fr;
 
-  scrollbar-color: @zesty-grey @zesty-light-grey;
-  scrollbar-width: thin;
+  //CLOSED MENU
+  &.NavClosed {
+    grid-template-columns: 60px 1fr;
 
-  @media (min-width: 2000px) {
-    & {
-      grid-template-columns: 170px 1fr;
-      font-size: 14px;
+    a[href="/content"],
```

Comment:
please use a class to target these

Reply:
@shrunyan these live inside NavClose class I am not 100% understanding the action on this one?

Reply:
These selectors use href attributes to target the elements. So we have to maintain this list going forward. Can we use a class instead?

Reply:
Since we are at shell top level I specifically used these selectors so I wouldn't have a cascading affect on other `a` tags